v3/plugins/redhat/vscode-camelk/0.0.9/meta.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-camelk
+version: 0.0.9
+type: VS Code extension
+displayName: Tooling for Apache Camel K
+title: Tooling for Apache Camel K
+description: VS Code support for Apache Camel K functionality
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/camel-tooling/vscode-camelk
+category: Language
+firstPublicationDate: '2019-07-10'
+spec:
+  containers:
+  - image: "eclipse/che-remote-plugin-camelk-0.0.9:latest"
```

Comment:
what is recommended here, use latest or next?

Reply:
for now (until we have sidecar theia without the theia runtime): 
next as it's upated automatically by che-theia CI

when Eclipse Che is tagged, we replace all `next` by the tagged che version. And this version of the registry is deployed to all che instances.

for example 7.3.0 will replace `next` by `7.3.0` and che server 7.3.0 will use plugin-registry 7.3.0

not sure I'm clear...